intel_owl/settings/logging.py

```diff
@@ -81,13 +81,50 @@
             "maxBytes": 20 * 1024 * 1024,
             "backupCount": 6,
         },
+        "rest_email_auth": {
+            "level": INFO_OR_DEBUG_LEVEL,
+            "class": "logging.handlers.RotatingFileHandler",
+            "filename": f"{DJANGO_LOG_DIRECTORY}/authentication.log",
+            "formatter": "stdfmt",
+            "maxBytes": 20 * 1024 * 1024,
+            "backupCount": 6,
+        },
+        "rest_email_auth_errors": {
+            "level": "ERROR",
+            "class": "logging.handlers.RotatingFileHandler",
+            "filename": f"{DJANGO_LOG_DIRECTORY}/authentication_errors.log",
```

Comment:
wrong file name

Reply:
rest_email_auth is used only in authentication app so it prints log in the same file